fmriprep/cli/run.py

```diff
@@ -168,9 +168,9 @@ def get_parser():
 
     # FreeSurfer options
     g_fs = parser.add_argument_group('Specific options for FreeSurfer preprocessing')
-    g_fs.add_argument('--fs-no-reconall', '--no-freesurfer',
```

Comment:
I'd keep this around for the time being, so that people can keep using their command line options. We can take it out in a version or two, once the docs have been changed for a bit.

@oesteban feel free to override me on this.

Reply:
What do you think about adding something like this in the description?
```
Note : `--no-freesurfer` is deprecated and will be removed in 1.2.
Use `--fs-no-reconall` instead.
```

Reply:
That seems reasonable to me.